Bottom: f2ea8d22d4a1c0b18dc8aff6b43329dce57cd708
Top:    b0ab9dc59aa949a005bd680a049c6fc1a6921d1e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 01:24:19 +0100

draw.c: Remove old commented debug printf


---

diff --git a/src/draw.c b/src/draw.c
index d351889..c041195 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1736,8 +1736,6 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
   Output.bgGC = gui->make_gc ();
   Output.pmGC = gui->make_gc ();
 
-  /*printf("\033[32mhid_expose_callback, s=%p %d\033[0m\n", &(SWAP_IDENT), SWAP_IDENT); */
-
   hid->set_color (Output.pmGC, "erase");
   hid->set_color (Output.bgGC, "drill");
